include/rocksdb/db.h

```diff
@@ -2083,12 +2083,9 @@ class DB {
 
   // Get the table properties of files per level.
   virtual Status GetPropertiesOfTablesForLevels(
-      ColumnFamilyHandle* /* column_family */,
-      std::vector<
-          std::unique_ptr<TablePropertiesCollection>>* /* levels_props */) {
-    return Status::NotSupported(
-        "GetPropertiesOfTablesForLevels() is not implemented.");
-  }
+      ColumnFamilyHandle* column_family,
+      std::vector<std::unique_ptr<TablePropertiesCollection>>*
+          levels_props) = 0;
```

Comment:
I have a slight preference for `props_by_level` (and would have preferred `GetPropertiesOfTablesByLevel`, but not a big deal. I should have suggested that before. "ForLevels" sounds like "WHERE level IN ..." instead of "GROUP BY level")

Reply:
Thanks, I appreciate a good naming suggestion! It is updated now.